### PR TITLE
Fix button alignment

### PR DESCRIPTION
**Pull request by Meesch**
After some noodling and css tutorials, I think the best option to make the buttons align nicely on all screens is to fit them a grid:

Screenshot 2022-06-14 at 16 03 00

Addresses #348

### CODE REVIEW

**Review by tymees — CHANGES_REQUESTED** [review]
The trajectory pages now look fine, but the others now look a bit weird:

image

There are two main issues at work here:
1) You've used a 4x1 grid for something that's not always 4 items, thus the browser adds a 'fake' column that takes up the rightmost space
2) By using a grid, all items must take the size it receives from te grid, leading to overly wide buttons

I don't know CSS grid well enough to propose a fix... I do know something that'll work using flexbox ;)

File: main/static/main/style.css [hunk 1]
```diff
@@ -39,10 +39,27 @@ form label {
...
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    column-gap: 5px;
+    text-align: center;
+}
+
+.button-grid > .button {
+    line-height: 1.5em;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
```
[the remark this answers is not in the document]
Comment (Meesch) [reply main/static/main/style.css:53]: yes, this is to make the buttons look good even when there are multiple lines of text:

Screenshot 2022-06-17 at 13 36 55

**Comment by Meesch** [comment]
Ok, update: I have made it so that there is now a dynamic number of columns, as requested. Personally, I like the wide buttons that take up the whole bottom, even in the extreme cases of two or even just one button:
Screenshot 2022-06-17 at 13 39 14

Screenshot 2022-06-17 at 13 39 37

But I realise this is a matter of taste and possibly convention, so I will defer to your judgement. Let me know what you want to do and I will do it!

File: main/static/main/style.css [hunk 2]
```diff
@@ -39,10 +39,28 @@ form label {
     display: inline-block;
 }
 
+.button-grid {
+    display: grid;
+    grid-auto-flow: column;
+    /* grid-auto-columns: min-content; */
```
Comment (tymees) [inline main/static/main/style.css:45]: This commented out line should be removed

**Review by tymees — APPROVED** [review]
If you remove that commented out line I think it should be fine to merge. 

> But I realise this is a matter of taste and possibly convention, so I will defer to your judgement. Let me know what you want to do and I will do it!

Personally, I don't like that style at all :P

But, it's now consistant across the app and working, so  ¯\\_(ツ)_/¯

**Pull request merged**